splinterd/src/main.rs

```diff
@@ -147,6 +147,8 @@ fn main() {
           "If set tls should accept all peer certificates")
         (@arg bind: --("bind") +takes_value
           "Connection endpoint for REST API")
+        (@arg database: --("database") +takes_value
```

Comment:
This will be included if the `"database"` feature is off. It should match the pattern below, like the `generate-certs` block.